site/pages/view-all-disruptions.page.tsx

```diff
@@ -523,6 +541,44 @@ const ViewAllDisruptions = ({
         } // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [filter]);
 
+    useEffect(() => {
+        const generatePdf = async () => {
+            if (downloadPdf) {
+                const parseDisruptions = exportDisruptionsSchema.safeParse(disruptionsToDisplay);
+                const blob = await pdf(
+                    <PDFDoc disruptions={parseDisruptions.success ? parseDisruptions.data : []} />,
+                ).toBlob();
+                saveAs(blob, "Disruptions.pdf");
+                setDownloadPdf(false);
+            }
+        };
+        generatePdf().catch(() => {
+            saveAs(new Blob(["There was an error. Contact your admin team"]), "Disruptions.pdf");
+        });
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [downloadPdf]);
+
+    useEffect(() => {
+        if (downloadExcel) {
+            generateExcel().catch(async () => {
+                await writeXlsxFile(["There was an error. Contact your admin team"], {
+                    schema: [{ column: "error", type: String, value: (objectData: string) => objectData }],
+                    fileName: "disruptions.xlsx",
+                });
+            });
+            setDownloadExcel(false);
+        }
+        // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
why was this disabled?

Reply:
eslint error "React Hook useEffect has a missing dependency: 'generateExcel'. Either include it or remove the dependency array" would be displayed where `generateExcel` dependency is not a state variable

Reply:
that is fine to include generateExcel you can remove the lint disable

Reply:
there are other examples above this code which is using the `disable-next-line` option in similar cases